spec/models/course_spec.rb

```diff
@@ -21,7 +21,6 @@
     it { is_expected.to have_many(:discussion_topics) }
     it { is_expected.to have_many(:forums).dependent(:destroy) }
     it { is_expected.to have_many(:lesson_plan_items).dependent(:destroy) }
-    it { is_expected.to have_many(:lesson_plan_milestones).dependent(:destroy) }
```

Comment:
Can you add a test similar to the one for `assessments`? i.e. `have_many ... through ...`
Do the same for the lesson plan events.